packages/transcribe/src/core/HtrCli.device.test.ts

```
@@ -0,0 +1,110 @@
+import HtrCli from './HtrCli';
+import * as utils from '@joplin/utils';
+
+jest.mock('@joplin/utils', () => ({
+    ...jest.requireActual('@joplin/utils'),
```

ESLint

[error] Expected indentation of 1 tab but found 4 spaces.

(`@typescript-eslint/indent`)

---

[error] Expected indentation of 1 tab but found 4 spaces.

(indent)